portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,86 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <title>Chen Xie</title>
+    <meta name="description" content="Chen Xie's personal website">
+    <meta name="author" content="Chen Xie">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+  
+    <link rel="stylesheet" href="css/styles.css">
+    <script src="js/script.js"></script>
   </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+  
+  <body onload = "myFetch()">
```

Comment:
Rename this method so that the reader can have a better understanding of what it does.